.travis.yml

```diff
@@ -22,7 +22,8 @@ env:
         # overidden underneath. They are defined here in order to save having
         # to repeat them for all configurations.
         - PYTHON_VERSION=3.6
-        - ASTROPY_VERSION=development
+        #- ASTROPY_VERSION=development
+        - ASTROPY_GIT='git+git://github.com/drdavella/astropy@asdf-entry-point#egg=astropy'
```

Comment:
Does astropy not use entry points to install the extension yet? Just wondering why you are pointing to your github repo.

Reply:
No, not yet. I'm going to submit that PR today also and then merge both of these at the same time.